ui/jobs/jobs.js

```diff
@@ -2396,6 +2408,14 @@ class Bars {
             if (m)
               this.buffTracker.onUseAbility(m.groups.id, log);
           }
+          // use of GP Potion
+          let cordialRegex = new RegExp(this.me + ':20(017FD|F5A3D|F844F|0420F|0317D):');
+          if (log.match(cordialRegex)) {
+            this.GpPotion = true;
```

Comment:
(Can you name this `gpPotion` or `usedCordial` or some variable name that doesn't start with a capital letter? I know the options are all capitals, and I'll fix that eventually, but other new variables should definitely be lower case where possible.)

Reply:
ah, I was confused. That's a mistake.